composer.json

```diff
@@ -32,7 +32,7 @@
 		"christophwurst/kitinerary-sys": "^0.2.1",
 		"ezyang/htmlpurifier": "4.16.0",
 		"gravatarphp/gravatar": "dev-master#6b9f6a45477ce48285738d9d0c3f0dbf97abe263",
-		"html2text/html2text": "^4.3.1",
+		"hamza221/html2text": "^1.0",
```

Comment:
Did you have to fork your own version? :scream_cat:

Reply:
yeah, I have a Pr open since march 
https://github.com/mtibben/html2text/pull/116